src/backend/aspen/api/schemas/samples.py

```diff
@@ -22,14 +24,20 @@ class SampleGisaidResponse(BaseResponse):
     status: str
 
 
+# TODO: add field to capture pangolin_last_updated (last_updated) once we add support for that
 class SampleLineageResponse(BaseResponse):
-    last_updated: Optional[datetime.datetime]
```

Comment:
we didn't move this over in the last migration to populate this table, are we dropping this field from the DB?

Reply:
i'm adding this field back in a future migration, but for now let's leave it off since that field currently doesn't exist on the SampleLineages table